homeassistant/components/tensorflow/image_processing.py

```diff
@@ -104,7 +106,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     try:
         # Display warning that PIL will be used if no OpenCV is found.
-        import cv2  # noqa: F401 pylint: disable=unused-import
+        import cv2  # noqa: F401 pylint: disable=unused-import, import-outside-toplevel
```

Comment:
This doesn't make sense. You import it at the top, so that's where it will raise already, but it's an allowed raise. This PR should not have the imports be moved around.